### PR TITLE
Added link to my blogpost.

### PR DESCRIPTION
**Pull request by TechnoAman**
Added link to my blogpost 
with url,title,img,author_name

Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -191,4 +191,9 @@
 - url: https://saarthakchaturvedi05.wordpress.com/2017/12/24/interview-with-long-term-open-source-contributors/
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
-  author_name: Saarthak Chaturvedi
\ No newline at end of file
+  author_name: Saarthak Chaturvedi
+  
+- url    : https://amantiwari123.wordpress.com/2017/12/27/my-journey-to-google-code-in-gci-2017-18/
```
Comment (akshatnitd) [inline _data/blogs.yml:196]: You have left extra spaces between `url` and `:` and similarly other key values. Make sure to follow the correct practice and change it the way other entries are mentioned.

File: _data/blogs.yml [hunk 2]
```diff
@@ -191,4 +191,9 @@
...
+  title  : My Journey to Google Code In ( GCI 2017-18).
+  img    : https://amantiwari123.files.wordpress.com/2017/12/img_20171227_140128.jpg?w=1272
```
Comment (akshatnitd) [inline _data/blogs.yml:198]: You don't need to provide a link here. Instead add an image of size 240*240px in the `img/blog/` folder and write the filename in the value for `img` key.

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Comment by akshatnitd** [comment]
Also, fill the Pull Request template correctly and follow good practices. You also need to provide a deployment link.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@TechnoAman  Please do the suggested changes

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Provide a Test-Link. 
> Follow the suggestions given by our mentors.

**Review by isuruAb — COMMENTED** [review]
Hello @TechnoAman , please provide a test link it speeds up the review process and it makes reviewers job easy. cheers

**Comment by ParthS007** [comment]
closing this as a duplicate of #553

**Comment by TechnoAman** [comment]
I have added the image after resize 240 x 240

**Pull request closed without merging**